docusaurus/src/remark/docsHeaderDecoration.js

```diff
@@ -25,6 +27,13 @@ const plugin = () => {
         const originalTitle = node.children[0].value;
         const originalId = node.data.hProperties.id;
 
+        const rawCDKVersion = getFromPaths(registryEntry, "packageInfo_[oss|cloud].cdk_version");
+        const syncSuccessRate = getFromPaths(registryEntry, "generated_[oss|cloud].metrics.cloud.sync_success_rate");
+        const usageRate = getFromPaths(registryEntry, "generated_[oss|cloud].metrics.[all|cloud|oss].usage");
```

Comment:
> Does the generated_oss metrics object actually have a .cloud.usage path? Or is this just a side-effect of the path unioning logic?

**Some context:**

1. `generated_oss` relates directly to the `generated` field in the OSS Registry
2. `generated_cloud` relates directly to the `generated` field in the Cloud Registry 

This is just a quirk of how we transform the 2 registries into one registry report for docs to use. So that we have 
the union of both catalogs.

Also the reason we have all vs oss vs cloud for metrics is simply because "we had the data in metrics"

We should always be using `all`, `all` should always have data in their respective registries.

**Answer**
generated_oss metrics does have a .cloud.usage path. But its really there as a backup, and `all` should always have data.

> Why are the usage metrics split up into separate paths like this?
Basically because we could and since we could may as well add them to make any future request easier.

Which became immediately useful when Natik asked if we could skip tests for low to none cloud usage.

> Does this mean we are always trying to show usage across all deployments if possible, but if not then only across cloud deployments, and if not that then only across oss deployments?
Yes

Reply:
Got it. If we expect `all` to usually always have data then this seems fine to me.